src/Views/Base.php

```diff
@@ -47,9 +47,11 @@ public function initialize(Request $request, Response $response, array $args = a
         $this->setArguments($args);
         $this->setVariable('args', $args);
         $this->setVariable('_env', $_ENV);
-        $this->setVariable('settings', $this->container->get('settings'));
+
         $revision = 'dev';
-        $projectDir = $this->container->get('settings')['project_root'];
+
+        // TODO: not sure what this points towards?
```

Comment:
That has to do with the way I handle deployments; the .revision file will contain the commit hash for the deployed version. Can be used to show/link to the commit, as cache buster for assets, etc.